src/components/Application/CSurprise.vue

```diff
@@ -275,34 +431,34 @@ export default {
 
   data () {
     return {
+      emptyForm,
+      fullForm,
       appData,
       components: this.$options.components,
       isEmptyFormPopulated: false,
       isFullFormPopulated: false,
-      emptyForm,
-      fullForm,
+      isPropertyClicked: false,
     }
   },
 
-  computed: {
-    app: {
-      get () {
-        return this.application
-      },
-
-      set (a) {
-        this.$emit('update:application', a)
-      },
-    },
+  created () {
+    console.log()
   },
 
-  created () {
+  mounted () {
     console.log()
   },
```

Comment:
I'm assuming you forgot to remove these `console.logs`?

Reply:
yup, gonna do a clean-up at some point later. Haven't forgotten about it.